pkg/controller/kogitodataindex/kogitodataindex_controller.go

```diff
@@ -73,7 +72,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	}
 
 	// Watch for changes to KogitoApp since we need their runtime images to check for labels, persistence and so on
-	err = c.Watch(&source.Kind{Type: &v1.BuildConfig{}}, &handler.EnqueueRequestForOwner{IsController: true, OwnerType: &appv1alpha1.KogitoApp{}})
```

Comment:
why removing it:?

Reply:
We are now watching `ConfigMap` instead. Take a look above. =)